packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -284,7 +285,7 @@ abstract contract StandardBridge is Initializable {
         require(_to != address(this), "StandardBridge: cannot send to self");
 
         emit ETHBridgeFinalized(_from, _to, _amount, _extraData);
-        (bool success, ) = _to.call{ value: _amount }(new bytes(0));
+        bool success = SafeCall.call(_to, gasleft(), _amount, hex"");
```

Comment:
Is there a good reason we didn't have ESC here before?

Reply:
No good reason, I feel like it should be here